include/deal.II/matrix_free/matrix_free.h

```diff
@@ -130,49 +130,55 @@ class MatrixFree : public Subscriptor
 
   /**
    * Collects the options for initialization of the MatrixFree class. The
-   * first parameter specifies the MPI communicator to be used, the second the
+   * parameter @p tasks_parallel_scheme specifies the
    * parallelization options in shared memory (task-based parallelism, where
    * one can choose between no parallelism and three schemes that avoid that
    * cells with access to the same vector entries are accessed
-   * simultaneously), the third with the block size for task parallel
-   * scheduling, the fourth the update flags that should be stored by this
-   * class.
+   * simultaneously), and the parameter @p tasks_block_size the block size for
+   * task parallel scheduling. The parameters @p mapping_update_flags,
+   * @p mapping_update_flags_boundary_faces, @p mapping_update_flags_inner_faces,
+   * and @p mapping_update_flags_faces_by_cells specify the update flags that
+   * should be stored by this class.
    *
-   * The fifth parameter specifies the level in the triangulation from which
+   * The parameter @p mg_level specifies the level in the triangulation from which
    * the indices are to be used. If the level is set to
-   * numbers::invalid_unsigned_int, the active cells are traversed, and
+   * `numbers::invalid_unsigned_int', the active cells are traversed, and
    * otherwise the cells in the given level. This option has no effect in case
    * a DoFHandler is given.
    *
-   * The parameter @p initialize_plain_indices indicates whether the DoFInfo
+   * The parameter @p store_plain_indices indicates whether the DoFInfo
    * class should also allow for access to vectors without resolving
    * constraints.
    *
-   * The two parameters `initialize_indices` and `initialize_mapping` allow
+   * The two parameters @p initialize_indices and @p initialize_mapping allow
    * the user to disable some of the initialization processes. For example, if
    * only the scheduling that avoids touching the same vector/matrix indices
    * simultaneously is to be found, the mapping needs not be
    * initialized. Likewise, if the mapping has changed from one iteration to
    * the next but the topology has not (like when using a deforming mesh with
    * MappingQEulerian), it suffices to initialize the mapping only.
    *
-   * The two parameters `cell_vectorization_categories` and
-   * `cell_vectorization_categories_strict` control the formation of batches
+   * The two parameters @p cell_vectorization_categories and
```

Comment:
I am not sure why these changes are necessary as they do not change the appearance at all (and we might now be changing back into a state we had before), but I am not opposed to it.

Reply:
I made these changes, since most of the parameters in the description used `@p`. Else I wouldn't have changed it.